src/systemTest/java/com/flightstats/hub/functional/callback/CallbackCache.java

```diff
@@ -1,6 +1,6 @@
-package com.flightstats.hub.callback;
+package com.flightstats.hub.functional.callback;
 
-import com.flightstats.hub.model.WebhookCallbackRequest;
+import com.flightstats.hub.functional.model.WebhookCallbackRequest;
```

Comment:
I think I understand the reasoning for moving models under `functional`, since right now they're only used there, but they could be useful as a package on their own. We could eventually use them + gson in the hub itself so we're not manually constructing JSON (which, iirc, we do!). It could also be used as a dependency in Paul's callback server and other teams have expressed an interest in using something like a model and client library that we provide.

Reply:
I thought about creating a reusable hub client. But am not sure if this is really worth the effort.
All other teams have been using their own hub clients for a while. It would be an additional unrewarding effort for teams to switch to new hub client without any obvious benefits. Given that Hub contract is not going to change anytime in the near future, I would say its not worth it. With Data lake work in progress, there is high possibility that hub may not be used in the future. Again this is something architecture will decide.

Reply:
That may be true, but moving the models into the functional tests when it's reasonable that they could be used elsewhere (like the callback server code in one of our other repositories) seems counterintuitive to me. Leaving them outside the functional tests makes it more apparent that we *do* have the beginnings of a shareable model/client library.

Reply:
ok moved the model package out of functional tests